### ISSUE TITLE
Design rationale on components and patterns

### ISSUE THREAD

**Issue by yatesn** [ONSdigital/design-system#1945]

**Comment by yatesn** [comment]
Not ready to do this yet, ideally need researcher

**Comment by yatesn** [comment]
Blocked by service manual

**Comment by daniellecorke** [comment]
Full UCD required - I will add to the UCD Jira board.

**Comment by daniellecorke** [comment]
On the UCD Board: https://jira.ons.gov.uk/browse/CU-66.

**Comment by daniellecorke** [comment]
Templates for documenting patterns and components:
[Template_for_design_system_components.docx](https://github.com/ONSdigital/design-system/files/15130099/Template_for_design_system_components.docx)
[Template_for_design_system_patterns.1.docx](https://github.com/ONSdigital/design-system/files/15130100/Template_for_design_system_patterns.1.docx)

**Comment by daniellecorke** [comment]
This issue is currently parked on the UCD Board.
